rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/selection/SelectionManager.java

```diff
@@ -88,8 +88,7 @@
 
     private RMCore rmcore;
 
-    private static final int SELECTION_THEADS_NUMBER = PAResourceManagerProperties.RM_SELECTION_MAX_THREAD_NUMBER
-            .getValueAsInt();
+    private static final int SELECTION_THEADS_NUMBER = PAResourceManagerProperties.RM_SELECTION_MAX_THREAD_NUMBER.getValueAsInt();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "SELECTION_THEADS_NUMBER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)